cura/resources/texts/whats_new/2.html

```diff
@@ -1,2 +1,2 @@
-<h4>Conditional statements for start and end G-code</h4>
-<p>It is now possible to include conditional statements such as If/Else statements when editing the start and end G-code in your machine settings. This allows you to set up much more complex printer automation that responds to things such as the status of your printer or user input. <a href="https://github.com/Ultimaker/Cura/wiki/Start-End-G%E2%80%90Code">Learn more about the possibilities on our wiki</a>.</p>
+<h4>Upcoming changes for USB-connected printing </H4>
+<p>In Cura 5.11, USB printing will no longer be enabled by default. New users will need to enable USB printing manually if they wish to use it. Anyone who printed using USB in Cura 5.10 can continue printing as normal. We have added the ‘Enable/Disable USB Printing’ setting to the preference menu to help with this change. Note that this refers to having your PC and printer directly linked via a USB cable, not printing with a USB hard drive.<p>
```

Comment:
Not really correct, this is a hidden setting in 5.10 and it doesn't show up in the menu